_attendees/myName.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
well done, can you remove this file?

Reply:
OK! I just see the issue. I will remove file and make new request